### ISSUE TITLE
Match Invite show ping

### ISSUE THREAD

**Issue by CrixUK** [ForFunLabs/ElevenPublicBugsAndAsks#303]
Hi, when someone sends a match request, would it be possible to see their ping next to their name so we can decide to accept or reject?

**Comment by Ken-Gilb** [comment]
Yes! Getting a ranked match with high lag is really difficult and feels a little unfair when you lose ELO because of it.
